MANIFEST.in

```diff
@@ -0,0 +1 @@
+recursive-include metaworld/envs/assets *
```

Comment:
Can you add a newline after this please?

Reply:
Sure, I'll wait on @tianheyu927 's review, and then commit any changes that are required.